core/frontend-stubs/maven-frontend-stub/src/test/java/org/eclipse/sw360/antenna/frontend/mojo/DependencyTest.java

```diff
@@ -36,8 +37,18 @@
  * she must also add it to the pom.xml of the maven-frontend-stub.
  */
 public class DependencyTest {
-    private static final Path PARENT_POM = Paths.get(DependencyTest.class.getProtectionDomain().getCodeSource().getLocation().getPath()).resolve("../../../../../pom.xml").normalize();
-    private static final Path MAVEN_FRONTEND_POM = Paths.get(DependencyTest.class.getProtectionDomain().getCodeSource().getLocation().getPath()).resolve("../../pom.xml").normalize();
+    private static Path PARENT_POM;
+    private static Path MAVEN_FRONTEND_POM;
+
+    static {
+        try {
+            MAVEN_FRONTEND_POM = Paths.get(DependencyTest.class.getProtectionDomain().getCodeSource().getLocation().toURI()).resolve("../../pom.xml").normalize();
```

Comment:
why behave `Paths.get(...getPath())` and `Paths.get(...toURI())` differently? What is the difference?

Reply:
ToUri: ![grafik](https://user-images.githubusercontent.com/44478897/59010146-902a2680-8830-11e9-8afa-5dce8a659a31.png)
GetPath: ![grafik](https://user-images.githubusercontent.com/44478897/59010296-16466d00-8831-11e9-9479-a8e52fd3c137.png)

Here the problem is described (the question explains the return of `getPath()` and the first answer that of `toUri()`: 
https://stackoverflow.com/questions/6164448/convert-url-to-normal-windows-filename-java